project/Component/NefMarkdown/NefMarkdown.swift

```diff
@@ -10,35 +10,77 @@ import Bow
 import BowEffects
 
 public struct Markdown {
-    private let render = Render()
+    public typealias MarkdownEnvironment = RenderMarkdownEnvironment<String>
```

Comment:
It is probably redundant, you can rename it to `Environment`, as it will be used as `Markdown.Environment`.

Reply:
Yes, I did it in the next PR ^^